.github/workflows/firefox-ios-update-icons.yml

```diff
@@ -0,0 +1,48 @@
+name: Update Icons
+
+on: 
+  workflow_dispatch:
```

Comment:
Do we want this to run manually or in a cron?

Reply:
We intend to run this Github Actions in a cron. Jackie and I discussed how to test this change on Github Actions before merging this PR. :)